reana_commons/api_client.py

```diff
@@ -67,7 +67,7 @@ def submit(self,
                prettified_cmd='',
                workflow_workspace='',
                job_name='',
-               cvmfs_mounts=False):
+               cvmfs_mounts='false'):
```

Comment:
Works fine, but such a value may be error prone.... An empty string would be a better default, like the rest? Might require changes in many places though...

Reply:
I think so too, empty string didn't work as it was passed as a boolean False in openapi calls.
Will look into it some more to find a better way.